src/graph/tuning.cc

```diff
@@ -167,7 +167,7 @@ static struct tuningModel tuning_model_0 {
     { 1.0, 0.8, 0.2, 1.0, 1.0, 0.3, 1.0, 0.1, 0.1, 0.2, 0.2, 0.1, 0.5, 1.0, 0.8, 0.8, 1.0, 0.9, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, },
   },
 
-  .llProtoRanges = {RCCL_LL_LIMITS_UNDEFINED},
+  .llProtoRanges = {{{RCCL_LL_LIMITS_UNDEFINED}}},
```

Comment:
I'm not sure if this resolves the warning. Please double-check. I suggest to comment out all these lines that assign to RCCL_LL_LIMITS_UNDEFINED and rely on the fact that the array will be value initialized to 0. In any case, this line of code will not work as it thinks it does. https://github.com/ROCm/rccl/commit/70cb1c2cd42233abf5fddf8d8b2dbfd7aa73853e